### PR TITLE
Add support for polling on the async worker

### PR DESCRIPTION
**Pull request by ocalvo**
This change allows the async worker to specify a custom polling function. This is useful in cases where the modem does not support callbacks.

### CODE REVIEW

**Comment by ocalvo** [comment]
@nijel Can you take a look?

File: gammu/asyncworker.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
 class GammuAsyncThread(gammu.worker.GammuThread):
     """Thread for phone communication."""
 
-    def __init__(self, queue, config, callback):
+    def __init__(self, queue, config, callback, pull_func):
```
Comment (nijel) [inline gammu/asyncworker.py:9]: Shouldn't the new parameter have a default value?

Comment (ocalvo) [reply gammu/asyncworker.py:9]: We provide all default parameters for GammuAsyncWorker, but for GammuAsyncThread there has never been defaults parameters, this is an internal implementation detail.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by ocalvo** [comment]
Thanks @nijel , can you publish a new version of python-gammu? We need this to integrate with homeassistant.

**Comment by nijel** [comment]
I'd like to have CI setup on GitHub for that (also to automate the upload to PyPI). Existing mixed CI setup with Travis and AppVeyor and manual uploads is annoying.

**Comment by nijel** [comment]
It's now uploaded: https://pypi.org/project/python-gammu/3.2.1/ (the description is now missing, but that should not matter).

**Pull request merged**